app/models/asset_field_type.rb

```diff
@@ -52,6 +53,11 @@ def mapping
 
   private
 
+  def asset_post_processing_data(metadata_hash)
```

Comment:
I changed the name so that it's no longer `format_data`, let me know if you still want it changed